src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -170,17 +171,17 @@ private function loadExternalFiles(RouteCollection $routeCollection)
     /**
      * Creates and adds a route for the given operation to the route collection.
      *
-     * @param RouteCollection $routeCollection
-     * @param string          $resourceClass
-     * @param string          $operationName
-     * @param array           $operation
-     * @param string          $resourceShortName
-     * @param string          $operationType
-     *
-     * @throws RuntimeException
+     * @param RouteCollection  $routeCollection
```

Comment:
You can drop this block, we now don't add PHPDoc when not adding information compared to the typehints. Just keep the `@throw` if appropriate.